src/tiled/mapdocumentactionhandler.h

```diff
@@ -104,6 +105,8 @@ public slots:
 
     void cropToSelection();
 
```

Comment:
I group similar functions together here, and I think there shouldn't be an empty line between `cropToSelection` and `autocrop` given their similar function.